packages/loader/driver-utils/src/runWithRetry.ts

```diff
@@ -66,10 +66,15 @@ export async function runWithRetry<T>(
             }
 
             if (progress.cancel?.aborted === true) {
+                logger.sendTelemetryEvent({
+                    eventName: `${fetchCallName}_runWithRetryAborted`,
+                    retry: numRetries,
+                    duration: performance.now() - startTime,
+                }, err);
                 throw new NonRetryableError(
-                    "runWithRetryAborted",
+                    "runWithRetry was Aborted",
                     OdspErrorType.fetchTimeout,
-                    { eventName: `runWithRetryAborted_${fetchCallName}`, driverVersion: pkgVersion },
+                    { driverVersion: pkgVersion },
```

Comment:
I'd add fetchCallName as a property, may make it easier to debug such events in the future.

Reply:
Here and the other two logs in this function

Reply:
And if you're making changes, I'd update the eventName to `..._abort` or `...aborted` (more concise)

Reply:
do you mean just on the error, not on the events?